abstracts/i_horse_race.sol

```diff
@@ -15,12 +17,53 @@ abstract contract IHorseRace {
     mapping(uint256 => Race) races;
     uint256 racesCount;
 
+    // MODIFIERS
+    modifier minHorses(uint256[] memory participatingHorses) {
+        require(participatingHorses.length >= 2, MIN_HORSES_ERROR_MESSAGE);
+
+        _;
+    }
+
+    modifier onlyOwner(string memory methodName) {
+        if (stringCompare(methodName, ADD_HORSE)) {
+            require(msg.sender == owner, ADD_HORSE_ERROR_MESSAGE);
+        } else if(stringCompare(methodName, ADD_JOCKEY)) {
+            require(msg.sender == owner, ADD_JOCKEY_ERROR_MESSAGE);
+        } else if(stringCompare(methodName, START_HORSE_RACE)) {
+            require(msg.sender == owner, START_RACE_ERROR_MESSAGE);
+        } else if(stringCompare(methodName, CANCEL_HORSE_RACE)) {
+            require(msg.sender == owner, CANCEL_RACE_ERROR_MESSAGE);
+        } else if (stringCompare(methodName, REFUND_REMOVE_BETS)) {
+            require(msg.sender == owner, REFUND_ERROR_MESSAGE);
+        }
+
+        _;
+    }
+
+    modifier invalidRaceId(uint256 raceId) {
+        require(
+            races[raceId].raceId != 0,
+            INVALID_RACE_ID_ERROR_MESSAGE
+        );
+
+        _;
+    }
+
+    modifier raceCompleted(uint256 raceId) {
```

Comment:
@pawneetdev  would make more sense if we name this function as **raceNotCompleted**, i was confused at start till i seen the function body

Reply:
My bad. I thought it for when the race has already complated.